src/backing/status/status.h

```diff
@@ -42,6 +42,14 @@ using Status = google::cloud::Status;
 template<typename T>
 using StatusOr = google::cloud::StatusOr<T>;
 
+// Converts the `StatusCode` to its underlying integer type.
+//
+// Useful for testing as well as defining error strings.
+constexpr std::underlying_type<StatusCode>::type StatusCodeToInt(
```

Comment:
Should the return type just be `int` since that's what the function name says it does?